src/azure-cli/azure/cli/command_modules/appservice/utils.py

```diff
@@ -128,10 +128,10 @@ def call(*args, **kwargs):
             while True:
                 try:
                     return func(*args, **kwargs)
-                except excpt_type as exception:  # pylint: disable=broad-except
+                except excpt_type:  # pylint: disable=broad-except
```

Comment:
Why we update this part, besides set a longer timeout?

Reply:
Just making the code more idiomatic. If the goal is to re-raise the exception, there's no point making a variable with it.

Also, if executing on Python2, the current code will lose the original stacktrace when re-raising the exception. 
But I'm not sure if az-cli supports running on Python 2 at all?